src/components/__tests__/User.test.js

```diff
@@ -0,0 +1,104 @@
+// User-test.js, both Login and registration
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { UserLogin } from '../Users/UserLogin';
+import { UserRegistration } from '../Users/UserRegistration';
+
+// describe what we are testing
+describe('Login Component', () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = shallow(<UserLogin />)
+    })
+ 
+    // make our assertion and what we expect to happen 
+    it('should render without throwing an error', () => {
+        expect(wrapper.find('.col.s8.form').exists()).toBe(true);
+    })
+
+    it('renders a email input', () => {
+        expect(shallow(<UserLogin />).find('#email').length).toEqual(1)
```

Comment:
why not use the wrapper variable you already declared since you are using shallow on both?